tests/testsuite.at

```diff
@@ -206,7 +206,7 @@ m4_define([NEGATIVE_TEST],[
 dnl Defines most relevant Souffle flag configurations for testing.
 dnl NOTE: This is the default configuration that can be overridden
 dnl using SOUFFLE_CONFS environment variable.
-m4_define([DEFAULT_CONFS], [[-j8],   dnl run interpreter in parallel
+m4_define([DEFAULT_CONFS], [[  -i -j8],   dnl run interpreter in parallel
```

Comment:
Indentation here could be nicer

Reply:
This was on purpose, unfortunately. Explicitly added 2 spaces so that when the testsuite is run, the options that show up match up. If it isn't added, weird macro expansion makes it not match.

Reply:
With the spaces:

```
Syntactic

  1: binary   -i -j8                                 ok
  2: binary   -c -j8                                 ok
```

Without the spaces:

```
Syntactic

  1: binary -i -j8                                   ok
  2: binary   -c -j8                                 ok
```

Reply:
I meant that the comments on the lines of testsuite.at should line up

Reply:
Oh, ok. I just jumped at the opportunity to justify those awful spaces. I'll fix this up.